tron/trondaemon.py

```diff
@@ -54,6 +55,7 @@ def setup_logging(options):
     # Show stack traces for errors in twisted deferreds.
     if options.debug:
         defer.setDebugging(True)
+    logging.getLogger = lru_cache(maxsize=None)(logging.getLogger)
```

Comment:
we should have a decent bit of explanation here - we'll want to:
* mention why we're doing this
* mention that this is a bit of a hack and that ideally we'd get rid of it
* link to the kubernetes-client issue that this attempts to workaround so that we can remove this whenever we get rid of this hack
* maybe point folks at misc/jolt#148 internally for more details?